modules/openapi-generator/src/main/resources/powershell-experimental/http_signature_auth.mustache

```diff
@@ -0,0 +1,162 @@
+{{>partial_header}}
+<#
+.SYNOPSIS
+Get the API key Id and API key file path.
```

Comment:
Let's use the same terminology as the HTTP signature spec:

> The "signature" authentication scheme is based on the model that the client must authenticate itself with a digital signature produced by either a private asymmetric key (e.g., RSA) or a shared symmetric key (e.g., HMAC). The scheme is parameterized enough such that it is not bound to any particular key type or signing algorithm. However, it does explicitly assume that clients can send an HTTP `Date` header.

The spec does not talk about "API keys".